README.md

```diff
@@ -9,6 +9,12 @@ If you need to run this website locally (development), you can run this command
 docker run --rm --volume="$PWD:/srv/jekyll" --volume="$PWD/vendor/bundle:/usr/local/bundle" -e JEKYLL_ENV="development" -it jekyll/jekyll:3.8 jekyll serve
 ```
 
+If you are under Windows with WSL or WSL2, you need to map a port as well. Then you can access using localhost (in this example, http://localhost:8080):
```

Comment:
why keeping the other version above and do not (always) the one with port 8080?

Reply:
Good point. Changed!